github/table_github_license.go

```diff
@@ -27,53 +26,67 @@ func tableGitHubLicense() *plugin.Table {
+			{Name: "conditions", Description: "An array of license conditions (include-copyright,disclose-source, etc).", Type: proto.ColumnType_JSON},
+			{Name: "description", Description: "The license description.", Type: proto.ColumnType_STRING},
+			{Name: "featured", Description: "If true, the license is 'featured' in the GitHub UI.", Type: proto.ColumnType_BOOL},
+			{Name: "implementation", Description: "Implementation instructions for the license.", Type: proto.ColumnType_STRING},
 			{Name: "key", Description: "The unique key of the license.", Type: proto.ColumnType_STRING},
-			{Name: "limitations", Description: "An array of limitations for the license (trademark-use, liability,warranty, etc).", Type: proto.ColumnType_JSON, Hydrate: tableGitHubLicenseGetData},
-			{Name: "permissions", Description: "An array of permissions for the license (private-use, commercial-use,modifications, etc).", Type: proto.ColumnType_JSON, Hydrate: tableGitHubLicenseGetData},
-			{Name: "url", Description: "The API url of the license.", Type: proto.ColumnType_STRING},
+			{Name: "limitations", Description: "An array of limitations for the license (trademark-use, liability,warranty, etc).", Type: proto.ColumnType_JSON},
+			{Name: "permissions", Description: "An array of permissions for the license (private-use, commercial-use,modifications, etc).", Type: proto.ColumnType_JSON},
+			{Name: "nickname", Description: "The customary short name of the license.", Type: proto.ColumnType_STRING},
+			{Name: "pseudo_license", Description: "Indicates if the license is a pseudo-license placeholder (e.g. other, no-license).", Type: proto.ColumnType_BOOL},
 		},
 	}
 }
 
-//// LIST FUNCTION
+type licenseRule struct {
+	Key         string
+	Label       string
+	Description string
+}
 
-func tableGitHubLicenseList(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
-	client := connect(ctx, d)
+type license struct {
+	SpdxId         string
+	Name           string
+	Key            string
+	Conditions     []licenseRule
+	Description    string
+	Featured       bool
+	Implementation string
+	Limitations    []licenseRule
+	Permissions    []licenseRule
+	Url            string
+	Nickname       string
+	PseudoLicense  bool
+}
 
-	type ListPageResponse struct {
-		licenses []*github.License
-		resp     *github.Response
-	}
+var listLicensesQuery struct {
+	Licenses []license `graphql:"licenses"`
+}
 
-	listPage := func(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
-		licenses, resp, err := client.Licenses.List(ctx)
-		return ListPageResponse{
-			licenses: licenses,
-			resp:     resp,
-		}, err
-	}
+var getLicenseQuery struct {
+	License license `graphql:"license(key: $key)"`
+}
 
-	listPageResponse, err := retryHydrate(ctx, d, h, listPage)
+// // LIST FUNCTION
```

Comment:
```suggestion
// LIST FUNCTION
```

Do we need two sets of `//`'s?

Reply:
Ha, originally it was 4 together (e.g. `//// HYDRATE FUNCTIONS`, my IDE appears to have inserted a space in between them - I don't mind using 4 or 2 is there a standard?

Reply:
If other tables use 4 `/`'s, then I'm ok to keep that style

Reply:
Ok, I'll make adjustment to remove the extraneous space (& dig into my IDE config to stop the behaviour of inserting spaces there...) shortly.